packages/angular/build/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-load("@npm//@angular/build-tooling/bazel/api-golden:index.bzl", "api_golden_test_npm_package")
+load("@@devinfra//bazel/api-golden:index_rjs.bzl", "api_golden_test_npm_package")
```

Comment:
Why is this `@@` prefixed instead of `@`?

Reply:
good catch. Unintentional; surprised it works 😄